assets/js/util/index.js

```diff
@@ -573,28 +586,26 @@ export const decodeHTMLEntity = ( str ) => {
  * Get the icon for a module.
  *
  * @param {string}  module                The module slug.
- * @param {boolean} blockedByParentModule Whether the module is blocked by a parent module.
  * @param {string}  width                 The icon width.
  * @param {string}  height                The icon height.
  * @param {string}  useClass              Class string to use for icon.
  *
  * @return {HTMLImageElement}             <img> tag with module icon.
  */
-export function moduleIcon( module, blockedByParentModule, width = '33', height = '33', useClass = '' ) {
-	if ( ! global._googlesitekitLegacyData ) {
+export function moduleIcon( module, width = '33', height = '33', useClass = '' ) {
+	if ( ! global._googlesitekitLegacyData || ! moduleMap.hasOwnProperty( module ) ) {
```

Comment:
This condition should no longer require `_googlesitekitLegacyData` since it isn't actually a dependency anymore 😄 
```suggestion
	if ( ! moduleMap.hasOwnProperty( module ) ) {
```

---

I'm starting to think this should simply be a `ModuleIcon` component, since it's basically a functional component now.
How about this:
```js
function ModuleIcon( { slug, className, width = 33, height = 33 } ) { ... }
// or even better
function ModuleIcon( { slug, width = 33, height = 33, ...props } ) { ... }
// and spread the rest into the inner component
```
That would make it much clearer to use as well rather than numeric positional parameters for width and height and `useClass` instead of `className` 😄 

There are only a handful of instances of this so it would be a simple refactor to apply as well.